docs/docs/guides/advanced/custom_RPC.md

```diff
@@ -18,27 +18,30 @@ In Web3.js 1.x, `web3.extend()` function could be used to add new JSON-RPC metho
 
 Following tutorial will guide you through the process of creating a custom plugin to extend the functionality of web3.js 4.x and add support for new RPC methods.
 
-#### Creating new RPC methods Plugin in 4 Steps
+## Creating new RPC methods Plugin in 4 Steps
+
+### Step 1: Setting Up Web3.js as a Peer Dependency and Creating a TypeScript Class
 
 1. First add web3.js as peer dependency in project´s `package.json` and create a TypeScript class for your plugin. This class should extend the `Web3Plugin` class provided by web3.js.
 
 :::info
 This will give your plugin access to [requestManager](/api/web3-core/class/Web3Context#requestManager) and [accountProvider](/api/web3-core/class/Web3Context#accountProvider).
-:::caution
-
+:::
 
 <Tabs groupId="prog-lang" queryString>
 
   <TabItem value="javascript" label="JavaScript"
   	attributes={{className: "javascript-tab"}}>
 
 ```javascript
-const { Web3PluginBase } = require('web3');
+const { Web3PluginBase } = require("web3");
```

Comment:
Thanks @SantiagoDevRel, 
However still double quot is used intensively, like here.
(By the way, I am referring to javascript ad typescript code only)

Reply:
Sorry, I missed the `advanced` folder to replace the `'`, it's done now.